database/seeds/ConferencesSeeder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Carbon\Carbon;
+use Faker\Factory as Faker;
+use Illuminate\Database\Seeder;
+use Laracasts\TestDummy\Factory;
+
+class ConferencesSeeder extends Seeder
+{
+    public function run()
+    {
+        Conference::truncate();
+
+        $faker = Faker::create();
+        $user_ids = collect(User::lists('id'));
+        $conference_names = collect([
+            'MegaAwesomeCon',
+            'SuperPHP',
+            'ActiveRecordCon',
+            'ConCon',
+            'GoodJobFest',
+            'TightenFest',
+            'UltraMegaCon',
+            'ArbysCon',
+        ]);
+
+        foreach (range(1, 30) as $i) {
+            $starts_at = $faker->dateTimeBetween('now', '3 years');
```

Comment:
Probably ok for now, I made sure the CFP always starts before the conference, but it could start 3 years before. These could definitely be smarter but it would be more work.

I did all this crap just so I could submit a PR that cleaned up your "+1000 years" sorting code, which I didn't even get around to, haha...